deployment/Procfile

```diff
@@ -2,5 +2,5 @@ manager-web-server: INTERNAL_PORT=$MANAGER_INTERNAL_PORT EXTERNAL_PORT=$MANAGER_
 manager: INTERNAL_PORT=$MANAGER_INTERNAL_PORT REMOTE_DEBUGGING_PORT=$MANAGER_REMOTE_DEBUGGING_PORT ./services/manager/start-web-environment.sh
 debug: REMOTE_DEBUGGER_PORT=$MANAGER_REMOTE_DEBUGGING_PORT PORT=$DEBUG_PORT npm --prefix services/debug start
 downloader: PORT=$DOWNLOADER_PORT npm --prefix services/downloader start
-speech-server: SOCKET_PATH=/tmp/radiodan-speech.sock simon-say
-speech: NO_AUTO_SPAWN=yes SOCKET_PATH=/tmp/radiodan-speech.sock npm --prefix services/speech start
+speech-server: SOCKET_PORT=$SPEECHD_PORT simon-say
+speech: NO_AUTO_SPAWN=yes SPEECHD_PPORT=$SPEECHD_PORT npm --prefix services/speech start
```

Comment:
`NO_AUTO_SPAWN=yes` vs `AUTO_SPAWN=no` ❓

Reply:
Why `SOCKET_PORT` when it's just a `PORT`?

Reply:
> NO_AUTO_SPAWN=yes vs AUTO_SPAWN=no?

Autospawn is the default behaviour. Instead of wondering if was `AUTO_SPAWN=no` or  `AUTO_SPAWN=false` or  `AUTO_SPAWN=0` and parsing them, I just thought setting the `NO_AUTO_SPAWN` to anything would do. Is that bad?

> Also, should this be PORT instead of PPORT?

Fixed.

> Why SOCKET_PORT when it's just a PORT?

Ok, `simon-say` accepts `PORT` now which is better.